### ISSUE TITLE
tpm2_createprimary: symbol lookup error

### ISSUE THREAD

**Issue by araff-r7** [tpm2-software/tpm2-tss#1831]
Hi, I am trying to bind Clevis to the TPM with this command on Ubuntu 20.04:

`sudo clevis luks bind -d /dev/nvme0n1p4 tpm2 '{"pcr_ids":"7"}'`

It previously worked on 18.04 with the tpm2-tools version 4.1.1 and tpm2-tss version 2.3.2, but now does not work on 20.04 with the latest versions of Clevis, tpm2-tools, and tpm2-tss. It gives the following error, which I cant find much info online about.

`tpm2_createprimary: symbol lookup error: tpm2_createprimary: undefined symbol: Esys_GetSysContext`

These are the commands I used to install from source:
`./configure --prefix=/usr --exec-prefix=/usr --disable-doxygen-man --disable-doxygen-doc`
`make`
`sudo make install`

It seems related to a change in the tpm-tss package, but this feature doesnt seem to be documented well so I can't be sure. Can you give me some tips on troubleshooting this? Thanks!

**Comment by williamcroberts** [comment]
Whats the tpm2-tss version? I think 20.04 is on version 2.3.2 and it requires 2.4:
```bash
git tag --contains ece734f40413b
2.4.0
```

**Comment by araff-r7** [comment]
I I tried compiling the latest version from the master branch.

**Comment by williamcroberts** [comment]
> I tried compiling the latest version from the master branch. Does the latest version not support 20.04?

tried compiling what from the master branch? tpm2-tss or tpm2-tools? Yes, Ubuntu 20.04 is supported. But if you use the tpm2-tss package from the ubuntu package manager, the version is 2.3.2 which is too old for current master tpm2-tools.

If you are building tpm2-tools from source code on branch master (anything post commit db83aee55b0e8), where I merged 4.2.X into master and you have a sub tpm2-tss package of below 2.4 you will get an error. For instance, I have tss2-esys 2.3.3 installed:
```bash
pkg-config --modversion tss2-esys
2.3.3
```

And when I build the tools, I get:
```bash
./bootstrap
./configure
configure: error: Package requirements (tss2-esys >= 2.4.0) were not met:
<snip>
```
What is the output of git describe on your tpm2-tools repo, assuming you cloned them with git?

**Comment by AndreasFuchsSIT** [comment]
I assume this is fixed.
If not, please mention it and I'll reopen.
